tests/test_visual.py

```diff
@@ -57,6 +59,41 @@ def test_histogram_plot():
         assert fig == mock_layout
 
 
+@patch("datasetinsights.stats.visualization.plots.go.Figure.add_trace")
+@patch("datasetinsights.stats.visualization.plots.go.Figure.update_yaxes")
+def test_model_performance_box_plot(mock_update, mock_add_trace):
+    mean_ap = [0.1, 0.2, 0.3]
```

Comment:
Do we need to create these many variables? As we are mocking the add trace, and testing hit count, can we reuse some of the variables?

Reply:
Added pytest fixture.